### DISCUSSION TITLE
New release for migrate/migrate

### DISCUSSION THREAD

**Discussion by ijatu-cloudbees** [golang-migrate/migrate#871] [General]
Team,

Thanks for the awesome work you are doing. The current image for migrate/migrate has been released in April 2022, it seems to have a number of vulnerabilities. Is there any plans of releasing a new version of the migrate/migrate images anytime soon ?

**Comment by amanbolat** [comment]
I just wanted to raise the same discussion or issue.
Not only the docker image, but also aws sdk has to be updated.

**Comment by ndsistsys** [comment]
Hi @dhui 
is this project actively maintained?
Thank you

**Comment by dhui** [comment]
Hi, sorry for the long time between releases but life has been busy.

I'll try to cut a new release in the next few weeks. In the past I've spent a large amount of time hand-writing release notes so that contributors are properly thanked but I don't have bandwidth for that anymore. I plan on switching to an automated release process so we'll have more frequent releases but the release notes will be lower in quality. I believe this strikes a better balance between my time, contributor attribution & recognition, and timely releases.

**Reply by scor** [reply]
While I agree it's important to give credit where it's due, keeping the tool up to date and secure and allowing for distribution of the work of all the contributors is just as important, if not more. Which automated release process are you planning to use? would it be possible to crowd source the work for the release notes to unblock you?

**Reply by dhui** [reply]
@scor Sorry for the delayed response! We're using [GoReleaser](https://github.com/goreleaser/goreleaser). Here's the [CI config](https://github.com/golang-migrate/migrate/blob/master/.github/workflows/ci.yaml) and [GoReleaser config](https://github.com/golang-migrate/migrate/blob/master/.goreleaser.yml)

Now that the releases are fully automated, I'll be able to release more frequently e.g. can cut a patch revision after security fixes are merged
